Examples/Run/Reconstruction/Common/RecCKFTracks.cpp

```diff
@@ -56,6 +57,8 @@ void addRecCKFOptions(ActsExamples::Options::Description& desc) {
       "Use track parameters smeared from truth particles for steering CKF");
   opt("ckf-truth-estimated-seeds", bool_switch(),
       "Use track parameters estimated from truth tracks for steering CKF");
+  opt("output-csv", bool_switch(),
```

Comment:
I think we don't have to define this but just add the `Options::addCsvWriterOptions(desc);` in the `runRecCKFTracks` function.

Reply:
Done